package.json

```diff
@@ -4,7 +4,8 @@
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "standard | snazzy && tap test/*.js"
+    "test": "standard | snazzy && tap test/*.js --no-coverage",
+    "coverage-ui": "tap test/*.js --coverage-report=html"
```

Comment:
Can you please do this in another PR? I think we should ignore ts, flow, esm, and jsx first. Once those are done we can keep coverage in as it would be faster anyway.